test/ae_mdw_web/controllers/activities_controller_test.exs

```diff
@@ -1000,5 +1000,138 @@ defmodule AeMdwWeb.ActivitiesControllerTest do
     end
   end
 
+  test "when ownership_only param given, it only returns owned by activities", %{conn: conn} do
+    account_pk = TS.address(0)
+    next_account_pk = TS.address(1)
+    contract_pk = TS.address(2)
+    account = Enc.encode(:account_pubkey, account_pk)
+    account_id = :aeser_id.create(:account, account_pk)
+    next_account_id = :aeser_id.create(:account, next_account_pk)
+    contract_id = :aeser_id.create(:contract, contract_pk)
+    height = 398
+    mbi = 2
+    kb_hash = TS.key_block_hash(0)
+    mb_hash = TS.micro_block_hash(0)
+    enc_mb_hash = Enc.encode(:micro_block_hash, mb_hash)
+
+    {:ok, aetx} =
+      :aec_spend_tx.new(%{
+        sender_id: account_id,
+        recipient_id: account_id,
```

Comment:
Done, it's not being used anyway because we're using the `Model.Field` records index to filter by ownership, but added it to display a more realistic scenario